monkey/infection_monkey/post_breach/actions/users_custom_pba.py

```diff
@@ -31,37 +32,49 @@ class UsersPBA(PBA):
     def __init__(self):
         super(UsersPBA, self).__init__(POST_BREACH_FILE_EXECUTION)
         self.filename = ''
+        self.command_list = []
         if not is_windows_os():
             # Add linux commands to PBA's
             if WormConfiguration.PBA_linux_filename:
                 if WormConfiguration.custom_PBA_linux_cmd:
                     # Add change dir command, because user will try to access his file
-                    self.command = (DIR_CHANGE_LINUX % get_monkey_dir_path()) + WormConfiguration.custom_PBA_linux_cmd
+                    cmd = (DIR_CHANGE_LINUX % get_monkey_dir_path()) + WormConfiguration.custom_PBA_linux_cmd
+                    self.command_list.append(cmd)
                     self.filename = WormConfiguration.PBA_linux_filename
+                    if self.filename not in cmd:  # PBA command is not about uploaded PBA file
+                        file_path = os.path.join(get_monkey_dir_path(), WormConfiguration.PBA_linux_filename)
+                        self.command_list.append(DEFAULT_LINUX_COMMAND.format(file_path))
```

Comment:
So now if the filename is not a part of the cmd, we run the file regardless?

Reply:
Yes